buildtools

```diff
@@ -82,11 +82,11 @@ buildnoplay() {
 
 companion() {
   title Building Companion...
-  any PlayApp
+  ant PlayApp
   if [ $? -eq 0 ]; then
     echo ""
     echo The companion is generated at:
-    echo $PWD/appinventor/build/buildserver/MIT AI2 Companion.apk
+    echo "$PWD"/build/buildserver/MIT AI2 Companion.apk
```

Comment:
I think @pavi2410 intended that the buildtools script was being run from the root of the repository rather than from the appinventor directory. For portability we might want to get `$(dirname $0)` rather than using $PWD since the script might be run from different locations.

Reply:
Yes, i ran that in the root directory, but i found that PWD gave a path that was /appinventor/appinventor/.. (2 times)

Reply:
You may also want to fix the path in the `extensions` command to match since it likely suffers the same problem.